### PR TITLE
allow to set the kubernetes log level

### PR DESCRIPTION
**Pull request by aojea**
allow to configure the verbosity of the logs on the kubernetes components.

Ref https://github.com/kubernetes-sigs/kind/pull/1750

Necessary to help debugging https://github.com/kubernetes/kubernetes/issues/127721

### CODE REVIEW

File: kubetest2-ec2/config/run-kubeadm.sh [hunk 1]
```diff
@@ -109,6 +109,9 @@ if [[ ${KUBEADM_CONTROL_PLANE} == true ]]; then
   KUBERNETES_VERSION=$(kubelet --version | awk '{print $2}')
   sed -i "s|{{KUBERNETES_VERSION}}|$KUBERNETES_VERSION|g" /etc/kubernetes/kubeadm-init.yaml
   sed -i "s|{{POD_CIDR}}|$POD_CIDR|g" /etc/kubernetes/kubeadm-init.yaml
+  KUBE_LOG_LEVEL=${KUBE_LOG_LEVEL:-4}
+  sed -i "s|{{POD_CIDR}}|$KUBE_LOG_LEVEL|g" /etc/kubernetes/kubeadm-init.yaml
```
Comment (dims) [inline kubetest2-ec2/config/run-kubeadm.sh:113]:
```suggestion
  sed -i "s|{{KUBE_LOG_LEVEL}}|$KUBE_LOG_LEVEL|g" /etc/kubernetes/kubeadm-init.yaml
```

File: kubetest2-ec2/config/run-kubeadm.sh [hunk 2]
```diff
@@ -109,6 +109,9 @@ if [[ ${KUBEADM_CONTROL_PLANE} == true ]]; then
...
+  sed -i "s|{{POD_CIDR}}|$KUBE_LOG_LEVEL|g" /etc/kubernetes/kubeadm-join.yaml
```
Comment (dims) [inline kubetest2-ec2/config/run-kubeadm.sh:114]:
```suggestion
  sed -i "s|{{KUBE_LOG_LEVEL}}|$KUBE_LOG_LEVEL|g" /etc/kubernetes/kubeadm-join.yaml
```

**Comment by aojea** [comment]
I see
```
+ [[ false == true ]]
+ sed -i 's|{{BOOTSTRAP_TOKEN}}|eac9lf.w8pfcb1pvey46iow|g' /etc/kubernetes/kubeadm-join.yaml
+ sed -i 's|{{KUBEADM_CONTROL_PLANE_IP}}|172.31.19.193|g' /etc/kubernetes/kubeadm-join.yaml
+ kubeadm join --v 10 --config /etc/kubernetes/kubeadm-join.yaml
I1027 09:48:36.855288   16364 join.go:421] [preflight] found NodeName empty; using OS hostname as NodeName
I1027 09:48:36.855337   16364 joinconfiguration.go:83] loading configuration from "/etc/kubernetes/kubeadm-join.yaml"
could not interpret GroupVersionKind; unmarshal error: error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{"KUBE_LOG_LEVEL":interface {}(nil)}
+ [[ false == true ]]
```

it has to replace the log level always or it will not work

**Comment by aojea** [comment]
pushed a new version

**Pull request closed without merging**